cli/config/config_test.go

```diff
@@ -1,7 +1,7 @@
-package mizu_test
+package config_test
 
 import (
-	"github.com/up9inc/mizu/cli/mizu"
+	config3 "github.com/up9inc/mizu/cli/config"
```

Comment:
why?

Reply:
what do you mean why? the number 3 is bothering you?